src/data/lovelace.ts

```diff
@@ -7,16 +7,18 @@ export interface LovelaceConfig {
 }
 
 export interface LovelaceViewConfig {
+  index?: number;
   title?: string;
   badges?: string[];
   cards?: LovelaceCardConfig[];
-  id?: string;
+  path?: string;
   icon?: string;
   theme?: string;
 }
 
 export interface LovelaceCardConfig {
-  id?: string;
+  index?: number;
```

Comment:
This won't support stacks editing, however, since it's frontend only, we can change it later.

Reply:
Yeah, we should have a parent_id for stacks or something

Reply:
`const [viewIndex, ...cardIndices] = obj_path`